icaruscode/CRT/CRTDecoder/DecoderICARUSCRT_module.cc

```diff
@@ -407,16 +421,25 @@ void crt::DecoderICARUSCRT::produce(art::Event& evt)
         destCh += recipe.direction; // increase or decrease the source
 
       }
-
+      allCRTdata.push_back(data);
     } // for all recipes
 
   } // for all input data
-
+  std::cout << "allCRTdata.size(): " << allCRTdata.size() << "\n";
   // move the data which is actually present in the final data product
   auto crtdata = std::make_unique<std::vector<icarus::crt::CRTData>>();
   for (icarus::crt::CRTData& crtDataElem: allCRTdata) {
     if (crtDataElem.fMac5 == 0) continue; // not a valid Mac5, data is not present
     crtdata->push_back(std::move(crtDataElem));
+    std::cout << "fMac5:  " << std::dec <<(int)crtDataElem.fMac5 << "\n";
+    std::cout << "fEntry: " << crtDataElem.fEntry << "\n";
+    std::cout << "fTs0:   " << crtDataElem.fTs0 << "\n";
+    std::cout << "fTs1:   " << crtDataElem.fTs1 << "\n";
+    std::cout << "\t[#ch]: ADC  ";
+    for(size_t i_c=0; i_c<32; ++i_c){
+      std::cout << "\t  ["<<std::setw(2)<<i_c<<"]: " <<std::setw(4)<< crtDataElem.fAdc[i_c];
+    }
+    std::cout << "\n------\n";
```

Comment:
I _request_ this output to be moved into a debug stream or removed: it's way too verbose for normal processing.
Example of the former:
```suggestion
    mf::LogTrace log { "DecoderICARUSCRT" };
    log << "fMac5:  " << std::dec <<(int)crtDataElem.fMac5 << "\n";
    log << "fEntry: " << crtDataElem.fEntry << "\n";
    log << "fTs0:   " << crtDataElem.fTs0 << "\n";
    log << "fTs1:   " << crtDataElem.fTs1 << "\n";
    log << "\t[#ch]: ADC  ";
    for(size_t i_c=0; i_c<32; ++i_c){
      log << "\t  ["<<std::setw(2)<<i_c<<"]: " <<std::setw(4)<< crtDataElem.fAdc[i_c];
    }
    log << "\n------";
```